server/routes/postQueryAnalysis.js

```diff
@@ -0,0 +1,47 @@
+const stopwordParser = require('../services/Parser')
+const parseIt = new stopwordParser()
+const express = require('express');
+const router = express.Router()
+
+let regex = /[^0-9a-z\s]/ig
+
+router.post('/', (req, res) => {
+    try{
+    let { query } = req.query
+    let queryArr = query.replace(regex, '').split(' ')
+    let data = []
+
+
+    for(let i = 0; i < queryArr.length; i++) {
+       let dateVal = parseIt.getDate(queryArr[i])
+       let mealtimeVal = parseIt.getMealtime(queryArr[i])
+       let stopwordVal = parseIt.getStopword(queryArr[i])
```

Comment:
Not related but let's say these were async functions. How would we change this to work?